src/base/perf_monitor.h

```diff
@@ -268,6 +276,14 @@ typedef enum
   PERF_SNAPSHOT_VISIBILITY_CNT
 } PERF_SNAPSHOT_VISIBILITY;
 
+typedef enum
+{
+  PERF_SERVED_COMPR_COUNT = 0,	/* per page type, this many pages have been compressed and served by page server */
+  PERF_SERVED_COMPR_RATIO,	/* per page type, the average ratio of pages compressed and served by page server */
+
+  PERF_SERVED_COMPR_CNT
```

Comment:
Nit : naming : PERF_SERVED_COMPR_CNT-> PERF_SERVED_COMPR_PAGE_TYPE_CNT (PERF_SERVED_COMPR_CNT can be confused with PERF_SERVED_COMPR_COUNT)